assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.spec.js

```diff
@@ -65,6 +65,44 @@ describe('Upgrade Flow - Upgrade Administration Server Controller', function ()
         should.exist(controller);
     });
 
+    describe('on controller creation', function () {
+        afterEach(function () {
```

Comment:
if the `initialResponse` is not reset after being modified on a test, it can affect other tests by reporting incorrect statuses due to the last modification, so we need to reset the `initialResponse` after each of those tests.

Also, the first test is not necessary, I just saw that those vars are already tested below. Ill remove it.